sympy/core/sympify.py

```diff
@@ -50,6 +52,23 @@ class CantSympify(object):
     """
     pass
 
+# Support for basic numpy datatypes
+
```

Comment:
two blank lines after previous class/def and no blank line following this comment,

Reply:
But its not fixed?

Reply:
Apologies - done now.